evm/v0.6/contracts/vendor/ENSResolver.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity ^0.6.0;
+
+abstract contract ENSResolver {
+  function addr(bytes32 node) public virtual view returns (address);
```

Comment:
It would be nice to try to stick to [the recommended order of modifiers](evm/v0.6/contracts/vendor/ENSResolver.sol): Visibility, Mutability, Virtual, Override, Custom modifiers.

Reply:
1f0f8f3